### PR TITLE
Add a Director-style WalkToBall strategy

### PR DESCRIPTION
**Pull request by ysims**
The WalkToBall strategy uses the FilteredBall info to either request a Task to walk to the ball or request a Task to turn on the spot (looking for the ball). 

This is the same behaviour as in the old system. Future work can involve aligning to the goals with the rotate around ball Task. 

Relies on #1037.

### CODE REVIEW

**Comment by ysims** [comment]
Hopefully good.

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
   "files.associations": {
     "*.role": "cmake",
     ".clang-tidy": "yaml",
-    ".Doxyfile": "properties"
+    ".Doxyfile": "properties",
+    "chrono": "cpp"
```
Comment (ysims) [inline .vscode/settings.json:22]:
```suggestion
    ".Doxyfile": "properties",
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   "files.associations": {
     "*.role": "cmake",
     ".clang-tidy": "yaml",
-    ".Doxyfile": "properties"
+    ".Doxyfile": "properties",
```
Comment (ysims) [inline .vscode/settings.json:21]:
```suggestion
    ".Doxyfile": "properties"
```

**Review by LachlanCourt — DISMISSED** [review]
LGTM

**Review by Crafty15 — APPROVED** [review]
LGTM 2

**Pull request merged**